source/mongorestore/mongorestore-examples.txt

```diff
@@ -345,14 +345,30 @@ namespace:
        You cannot restore the ``system.buckets.weather.bson`` file
        by itself. Attempting to do so results in an error.
 
+Use mongorestore --oplogReplay and mongodump --oplog 
+----------------------------------------------------
+
```

Comment:
ditto my previous comments about phrasing this as a way to capture/restore oplog entries that occur during the dump